optimize/self-managed/optimize-deployment/configuration/logging.md

```diff
@@ -6,11 +6,11 @@ description: "Camunda Optimize provides logging facilities that are preconfigure
 
 Camunda Optimize provides logging facilities that are preconfigured to use
 _INFO_ logging level which provides minimal output of information in log files.
-This level can be adjusted using the `environment-logback.xml` configuration file.
+This level can be adjusted using the `config/log4j2.xml` configuration file.
```

Comment:
This should still be in the format `environment-log4j2.xml`.
The code supports it